screwdriver/screwdriver.go

```diff
@@ -38,16 +43,33 @@ type api struct {
 	buildID string
 	baseURL string
 	token   string
-	client  *http.Client
+	client  *retryablehttp.Client
 }
 
 // New returns a new API object
 func New(buildID, url, token string) (API, error) {
+	// read config from env variables
+	if strings.TrimSpace(os.Getenv("LOGSERVICE_SDAPI_TIMEOUT_SECS")) != "" {
```

Comment:
move it out of the function to where `httpTimeout` is defined at top ?
Maybe just call, LOGSERVICE_HTTP_TIMEOUT_SECS and have it common for store upload as well.

Reply:
api and store can have different timeouts, so kept it separate. Top declarations are default.